### ISSUE TITLE
Java Lamda Explanation

### ISSUE THREAD

**Issue by dschandrasekara** [MLSA-Mehran-UET/Learn-to-code-java#51]
I really like to this project, i couldn't find any examples of java lamda.
If you are ok, shall create PR for it ?

**Comment by owais4321** [comment]
@dschandrasekara  yeah start working on them and create PR when you are done 
Thanks Happy Coding <3

**Comment by dschandrasekara** [comment]
Yes sure, i will start working on this, feel free plz assign this issue.

**Comment by dschandrasekara** [comment]
Created PR - https://github.com/MLSA-Mehran-UET/Learn-to-code-java/pull/87 and please help to review and appreciate ur feedback.
